Bluejay.asm

```diff
@@ -1712,194 +1732,218 @@ scheduler_start:
 	; Increment Scheduler Counter
 	inc Scheduler_Counter
 
-;******************  128 ms scheduler *******************
-;***********  For PWM temperature limiting **************
-scheduler_128ms_switch_case:
-	; Apply 128ms mask to Scheduler_Counter (2 lowest bits)
+	; Choose between odd or even steps
 	mov A, Scheduler_Counter
-	anl A, #03h
+	jb ACC.0, scheduler_steps_1357
 
-scheduler_128ms_switch_case_32ms:
-	cjne A, #1, scheduler_128ms_switch_case_64ms
+scheduler_steps_0246:
```

Comment:
maybe we can rename this to `scheduler_steps_even` and  `scheduler_steps_odd` for better clarity?

Reply:
done